### DISCUSSION TITLE
Add another tab (enterneth tab)

### DISCUSSION THREAD

**Discussion by TheNormanJames** [responsively-org/responsively-app#640] [Ideas]
Yes, I know the "function" of the tool.

But I would like to add another feature. And is a "project preview tab". 
For example. I'm working on a project with adobe XD, and the "design" link is online. So To watch or see this design I have to go to Google and open the link of the project.
So it could be great if you could add these features. to become "responsively" as All-in-one tool. 
So I could preview or watch the design in the same "responsively app" without going to the "internet browser" or switching between apps...

Thanks, guys... 💪

**Comment by manojVivek** [comment]
@NormanJaimes Thanks for opening this. 

We have plans to add design mock-up preview side-by-side to the browser preview. That should make this even better. 
I'll just create an issue from this discussion to track that.
